### ISSUE TITLE
SD card read/write extremely slow with version 2.0.2

### ISSUE THREAD

**Issue by boeckhoff** [espressif/arduino-esp32#6475]
### Board

Adafruit ESP-32 Feather

### Device Description

Feather is connected to a SD card module through jumper wires 

### Hardware Configuration

nothing

### Version

v2.0.2

### IDE Name

Arduino IDE (up to date)

### Operating System

Windows 10

### Flash frequency

80Mhz

### PSRAM enabled

no

### Upload speed

921600

### Description

When using the new version of the core (2.0.2) I get extremely slow SD card read/write speeds (1048576 bytes read for 829442 ms !!). With the same hardware and the 1.0.6 core I get 2640 ms for the same data read.

I reproduced this multiple times with both the Arduino IDE and platformio and 2 types of SD card modules.

### Sketch

[SD-Test sketch for ESP32](https://github.com/espressif/arduino-esp32/blob/master/libraries/SD/examples/SD_Test/SD_Test.ino)

### Debug Message

With Core@2.0.2

```
[..other info..]
1048576 bytes read for 829442 ms
1048576 bytes written for 216050 ms
```

With Core@1.0.6

```
[..other info..]
1048576 bytes read for 2640 ms
1048576 bytes written for 9652 ms
```

### I have checked existing issues, online documentation and the Troubleshooting Guide

- [X] I confirm I have checked existing issues, online documentation and Troubleshooting guide.

**Comment by mrengineer7777** [comment]
Duplicate of https://github.com/espressif/arduino-esp32/issues/6295. Should be fixed in arduino-esp32 v2.0.3. See also workarounds in https://github.com/espressif/arduino-esp32/issues/6295.  v2.0.3 should be released soon.

Duplicate https://github.com/espressif/arduino-esp32/issues/6338

**Comment by VojtechBartoska** [comment]
closing as the solution, thanks for linking a solution @mrengineer7777!
